pkg/opentelemetry/output.go

```diff
@@ -190,3 +192,14 @@ func (o *Output) dispatch(entry metrics.Sample) error {
 func normalizeMetricName(cfg Config, name string) string {
 	return cfg.MetricPrefix.String + name
 }
+
+func normalizeUnit(vt metrics.ValueType) string {
+	switch vt {
+	case metrics.Time:
+		return "ms"
+	case metrics.Data:
+		return "By"
```

Comment:
```suggestion
		return "byte"
```
Is it on purpose?

Reply:
Yes, it's by purpose. OpenTelemetry follows the UCCM standard where bytes is `By` see https://github.com/open-telemetry/opentelemetry-specification/issues/2973